src/osdag/design_type/compression_member/Column.py

```diff
@@ -1105,27 +1116,37 @@ def design_column(self):
                         self.optimum_section_cost_results[self.cost][j] = k
                         list_2.pop(0)
                         break
-        else:
-            logger.warning("The section(s) defined for performing the column design is/are not selected based on the selected Inputs and/or "
-                           "Design Preferences")
-            logger.error("Cannot compute!")
-            logger.info("Change the inputs provided and re-design.")
-            self.design_status = False
-            self.design_status_list.append(self.design_status)
+        #else:
+        #    logger.warning("The section(s) defined for performing the column design is/are not selected based on the selected Inputs and/or "
+        #                   "Design Preferences")
+        #    logger.error("Cannot compute!")
+        #    logger.info("Change the inputs provided and re-design.")
+        #    self.design_status = False
+        #    self.design_status_list.append(self.design_status)
             # print(f"design_status_list{self.design_status_list}")
```

Comment:
Again, looks like needed code.

Reply:
This else is part of the "if" condition I mentioned, else here refers when no section is selected as input.